### ISSUE TITLE
[Exception][IDEA 2023.2.1] java.lang.NullPointerException: it.text must not be null

### ISSUE THREAD

**Issue by malvemor** [Codium-ai/codiumai-jetbrains-release#93]
# User Description
Once installed, I tried to use it in an @service class.

# Stack Trace
```
java.lang.NullPointerException: it.text must not be null
	at ai.codium.psi.ImportsContentService$getContextCodium$contextCodium$1.invoke(ImportsContentService.kt:159)
	at ai.codium.psi.ImportsContentService$getContextCodium$contextCodium$1.invoke(ImportsContentService.kt:159)
	at kotlin.text.StringsKt__AppendableKt.appendElement(Appendable.kt:85)
	at kotlin.collections.CollectionsKt___CollectionsKt.joinTo(_Collections.kt:3347)
	at kotlin.collections.CollectionsKt___CollectionsKt.joinToString(_Collections.kt:3364)
	at kotlin.collections.CollectionsKt___CollectionsKt.joinToString$default(_Collections.kt:3363)
	at ai.codium.psi.ImportsContentService.getContextCodium(ImportsContentService.kt:159)
	at ai.codium.psi.ImportsContentService.getImportsContent(ImportsContentService.kt:34)
	at ai.codium.AnalysisManagerKt.runAnalysis(AnalysisManager.kt:52)
	at ai.codium.AnalysisManagerKt.runAnalysis$default(AnalysisManager.kt:33)
	at ai.codium.CodiumInlayProviderBase.addCodiumTestsGenerationInlay$lambda$0(CodiumInlayProviderBase.kt:75)
	at com.intellij.codeInsight.hints.presentation.PresentationFactory$referenceOnHover$hovered$1.invoke(PresentationFactory.kt:262)
	at com.intellij.codeInsight.hints.presentation.PresentationFactory$referenceOnHover$hovered$1.invoke(PresentationFactory.kt:258)
	at com.intellij.codeInsight.hints.presentation.PresentationFactory$onClick$2.invoke(PresentationFactory.kt:236)
	at com.intellij.codeInsight.hints.presentation.PresentationFactory$onClick$2.invoke(PresentationFactory.kt:234)
	at com.intellij.codeInsight.hints.presentation.OnClickPresentation._init_$lambda$0(OnClickPresentation.kt:17)
	at com.intellij.codeInsight.hints.presentation.OnClickPresentation.mouseClicked(OnClickPresentation.kt:21)
	at com.intellij.codeInsight.hints.presentation.StatefulPresentation.mouseClicked(StatefulPresentation.kt:96)
	at com.intellij.codeInsight.hints.presentation.StaticDelegatePresentation.mouseClicked(StaticDelegatePresentation.kt:51)
	at com.intellij.codeInsight.hints.presentation.InsetPresentation$mouseClicked$1.invoke(InsetPresentation.kt:53)
	at com.intellij.codeInsight.hints.presentation.InsetPresentation$mouseClicked$1.invoke(InsetPresentation.kt:52)
	at com.intellij.codeInsight.hints.presentation.InsetPresentation.handleMouse(InsetPresentation.kt:48)
	at com.intellij.codeInsight.hints.presentation.InsetPresentation.mouseClicked(InsetPresentation.kt:52)
	at com.intellij.codeInsight.hints.presentation.RecursivelyUpdatingRootPresentation.mouseClicked(RecursivelyUpdatingRootPresentation.kt:69)
	at com.intellij.codeInsight.hints.LinearOrderInlayRenderer.mouseClicked(LinearOrderInlayRenderer.kt:97)
	at com.intellij.codeInsight.hints.presentation.listeners.InlayEditorMouseListener

<TRIMMED STACKTRACE>

e$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:766)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:763)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:685)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:633)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:588)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

# Other Information
```
=== About ===
Build version: IntelliJ IDEA 2023.2.1 Build: #IU-232.9559.62 August 23, 2023
Theme: Dark
JRE: 17.0.8+7-b1000.8, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.8, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [Python (232.9559.62), Big Data Tools (232.9559.34), Terraform and HCL (232.8660.88), Flink (232.9559.34), Spark (232.9559.62), Pokemon Progress (2.1.1), SonarLint (9.0.0.75308), Nx Console (1.12.0), CodiumAI - Meaningful tests powered by TestGPT-1 and GPT-3.5&4 (0.6.02), Big Data File Viewer (232.8660.206), Rainbow Brackets (2023.3.4), Kubernetes (232.9559.34), Big Data Tools Core (232.9559.62), Zeppelin (232.8660.206), Remote File Systems (232.9559.62), Metastore Core (232.9559.62), AWS Toolkit (1.76-232), Kafka (232.9559.62), Package Search (232.9559.10)]
Disabled plugins:[Nx Console Idea (0.59.0)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/vfiT81m5) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @malvemor 👋

Thanks for reporting. We are working on a fix for this!

Best regards,
Alex

**Comment by ArtsiomCh** [comment]
fixed in 0.6.03
